Packs/PhishUp/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "PhishUp",
+    "description": "PhishUp prevents phishing attacks, protects your staff and your brand with AI",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Mehmet Cagri Caliskan",
+    "url": "",
```

Comment:
Do you also have a URL that users can visit in case they come across issues and need help? support portal or landing page?